packages/frontend/src/components/UserSettings/AccessTokensPanel/TokensTable.tsx

```diff
@@ -28,22 +38,87 @@ import {
 import MantineIcon from '../../common/MantineIcon';
 
 const TokenItem: FC<{
-    token: ApiPersonalAccessTokenResponse;
-    setTokenToDelete: Dispatch<
-        SetStateAction<ApiPersonalAccessTokenResponse | undefined>
-    >;
+    token: PersonalAccessToken;
+    setTokenToDelete: Dispatch<SetStateAction<PersonalAccessToken | undefined>>;
 }> = ({ token, setTokenToDelete }) => {
-    const { description, expiresAt } = token;
+    const { description, expiresAt, rotatedAt, lastUsedAt, uuid } = token;
 
     return (
         <>
             <tr>
                 <Text component="td" fw={500}>
                     {description}
                 </Text>
-
                 <td>
-                    {expiresAt ? formatDate(expiresAt) : 'No expiration date'}
+                    <Group align="center" position="left" spacing="xs">
+                        <span>
+                            {expiresAt
+                                ? formatDate(expiresAt)
+                                : 'No expiration date'}
+                        </span>
+                        {rotatedAt && (
+                            <Tooltip
+                                withinPortal
+                                position="top"
+                                maw={350}
+                                label={`Last rotated at ${formatTimestamp(
+                                    rotatedAt,
+                                )}`}
+                            >
+                                <MantineIcon
+                                    icon={IconInfoCircle}
+                                    color="gray.6"
+                                    size="md"
+                                />
+                            </Tooltip>
+                        )}
+                    </Group>
+                </td>
+                <td>
+                    {lastUsedAt ? (
+                        <Tooltip
+                            withinPortal
+                            position="top"
+                            maw={350}
+                            label={formatTimestamp(lastUsedAt)}
+                        >
+                            <Text>{formatDate(lastUsedAt)}</Text>
+                        </Tooltip>
+                    ) : (
+                        <Text color="gray.6">Never used</Text>
```

Comment:
will this cause confusion for existing tokens ? since this lastUsedAtt is new and nullable, existing tokens will all show `never used` (until they are used again)

Reply:
@rephus Fair point 💡  Options:
- keep the same
- don't show any value. 
- have a condition with a hardcoded date `if was created before 14 Nov 2024 and used date is null don't show any value otherwise "Never used"` 
Any preference or other option comes to mind? I'm inclined to the second.

Reply:
Went with second option